staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flowcontrol
+
+import (
+	"reflect"
+	"testing"
+
+	flowcontrol "k8s.io/api/flowcontrol/v1beta1"
+)
+
+func Test_configController_generatePatchBytes(t *testing.T) {
+	type args struct {
```

Comment:
Why is this level of structure here?  Why not just let the test case struct directly have a `flowcontrol.FlowSchemaCondition` field?

Reply:
I fixed this now. Please check.